IAM/openid/README.md

```diff
@@ -0,0 +1,5 @@
+# OpenID Connect Configuration File for SystemLink
```

Comment:
What are these IAM files? Did you mean to submit them here? You can have files within the handbook chapter and link to them. They'll show up as file downloads.

Reply:
So I want to begin establishing a pattern where the handbook provides examples in addition to pure documentation. I do want to get feedback on here to place them. I put them in their current location to get feedback. 

Would you recommend a structure more like 
```
handbook-
      |
       - openid-connect
                     |
                      - openid-connect.md
                      - example
                              |
                               - foo.povider
                               - foo.client
                               - foo.conf
```
I'll probably move all the images in the chapter to the img directory.

Reply:
Yeah, I had imagined something along those lines. Users can still browse the repo (or we could provide links to the repo), but those files will also be included in the mkdocs site and show up as file downloads when linked to.

Reply:
Changed!

I also changed the .pages file at the root of the handbook to use `nav` rather than `arrange` because arrange is to be [deprecated](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin#arrange-pages). I also added a .pages file in the directory containing the example config files so that its README.md file doesn't show up in handbook TOC.